operator/samples/pilot-advanced-override.yaml

```diff
@@ -1,23 +1,21 @@
 apiVersion: install.istio.io/v1alpha1
 kind: IstioOperator
 spec:
-  trafficManagement:
-    enabled: true
-    components:
-      proxy:
-        enabled: false
-      pilot:
-        k8s:
-          overlays:
-            - kind: Deployment
-              name: istio-pilot
-              patches:
-                - path: spec.template.spec.containers.[name:discovery].args.[30m]
-                  value: "60m" # OVERRIDDEN
-                - path: spec.template.spec.containers.[name:discovery].ports.[containerPort:8080].containerPort
-                  value: 8090 # OVERRIDDEN
-            - kind: Service
-              name: istio-pilot
-              patches:
-                - path: spec.ports.[name:grpc-xds].port
-                  value: 15099 # OVERRIDDEN
+  components:
+    proxy:
+      enabled: false
```

Comment:
Why do we have this? What does this even do?

Reply:
Just keep original settings.